lib/engine/game/g_1817.rb

```diff
@@ -34,6 +34,31 @@ class G1817 < Base
       GAME_DESIGNER = 'Craig Bartell, Tim Flowers'
       GAME_PUBLISHER = Publisher::INFO[:all_aboard_games]
       GAME_INFO_URL = 'https://github.com/tobymao/18xx/wiki/1817'
+
+      # @todo: this needs purchase of the 8 train
+      GAME_END_CHECK = { bankrupt: :immediate }.freeze
+
+      attr_reader :loans_taken, :loans_available, :loan_increments
+
+      def bankruptcy_limit_reached?
+        @players.reject(&:bankrupt).one?
+      end
+
+      def init_bank
+        @loans_taken = 31
+        @loans_available = 14 * 5
+        @loan_increments = 100
```

Comment:
maybe we should have a loan object 

loan_increments is really the loan value?